### PR TITLE
Actions workflow

### PR DESCRIPTION
**Pull request by claireh93**
Incorporated GH actions workflow from Phylo package.

### CODE REVIEW

**Comment by richardreeve** [comment]
@claireh93 I've added a deployment key for the docs now they are happening in a github action. Hopefully that will work!

**Review by richardreeve — COMMENTED** [review]
V minor changes - otherwise looks good to me!

**Comment by richardreeve** [comment]
Once you've merged this, you should be able to go to #33 and just make a comment that says:
```
@JuliaRegistrator register
```
And we'll get a new version!

**Comment by richardreeve** [comment]
Actually, we'll probably want to wait for CompatHelper to trigger and merge those PRs first now I think about it!

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Diversity"
 uuid = "d3d5718d-52de-57ab-b67a-eca7fd6175a4"
-version = "0.5.1"
 author = ["Richard Reeve <Richard.Reeve@glasgow.ac.uk>"]
+version = "0.5.1"
```
[the remark this answers is not in the document]
Comment (richardreeve) [reply Project.toml]: @claireh93 I've merged these changes just to get the testing to complete. If you're happy, then feel free to merge the PR and then we can wait for compathelper to kick in and then release a new version.

**Comment by claireh93** [comment]
Thanks for merging those @richardreeve! Seems to be a bit of a problem with DataFrames but I'll try to sort that today and merge those changes.

**Comment by richardreeve** [comment]
Ah, I can probably help with that this afternoon.

File: Project.toml [hunk 2]
```diff
@@ -15,16 +15,16 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-julia = "^1"
 AxisArrays = "^0.4"
-DataFrames = "^0.20"
+DataFrames = "^0.21"
```
[the remark this answers is not in the document]
Comment (richardreeve) [reply Project.toml]: This is now wrong unfortunately! `innerjoin()` doesn't exist in DataFrames v0.20.x. It should be:
```
DataFrames = "0.21,0.22"
```

**Comment by richardreeve** [comment]
@claireh93 I don't know if you saw my comment further up, but the DataFrames compat now needs to be changed to just `"0.21,0.22"` because 0.20.x doesn't include innerjoin! Sorry...

**Pull request merged**